src/XenCert/StorageHandler.py

```diff
@@ -652,7 +670,19 @@ def BlockUnblockPaths(self, blockOrUnblock, script, noOfPaths, passthrough):
                 raise Exception("   - The path block/unblock utility returned an error: %s." % stderr)
             return stdout
         except Exception, e:            
-            raise e        
+            raise e
+
+    def WaitManualBlockUnblockPaths(self):
+        try:
+            cmd = [self.storage_conf['pathHandlerUtil']]
+            (rc, stdout, stderr) = util.doexec(cmd, '')
+            XenCertPrint(
```

Comment:
Seems this function is for block/unblock the paths.
From the logs is a little bit confused:
`The path manually block/unblock utility returned ... `  --> `The manually block/unblock path utility returned ...`